Tests/Dumper/TranslationDumperTest.php

```diff
@@ -153,7 +154,7 @@ public function setUp()
         $container = $this->getContainer();
 
         $this->target     = sys_get_temp_dir() . '/bazinga/js-translation-bundle';
-        $this->filesystem = $container->get('filesystem');
```

Comment:
the filesystem is not in the container?

Reply:
not by default, at least.

Reply:
⬆️ what he said. I assume in Symfony 4 or later the `filesystem` service became private (along with many other services).